packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -34,7 +35,10 @@ export class TestAccount {
 	 * @param {Page} page Page object.
 	 * @param {string} [url] URL to expect once authenticated and redirections are finished.
 	 */
-	async authenticate( page: Page, { url }: { url?: string | RegExp } = {} ): Promise< void > {
+	async authenticate(
+		page: Page,
+		{ url, waitUntilStable }: { url?: string | RegExp; waitUntilStable?: boolean } = {}
```

Comment:
This change and the accompanying changes in Line 57-60 are important for stable loading of specs that rely on interaction in the My Home page.

Because Calypso loads so slowly but Playwright executes at lightning speed, we can often end up in a situation where Calypso "swallows" interactions on the page. To work around this issue, we can wait until the page loads entirely, but on Simple sites, this is often in excess of 15-20 seconds.

However, the Sidebar component is rendered and ready to interact towards the middle-later portions of the loading process. Waiting for the Sidebar component to become ready is suitable 98% of the time.